specimen/wildlife-sounds-bats-config.xml

```diff
@@ -0,0 +1,224 @@
+                <path>gatheringEvent.altitude</path>
+            </field>
+            <!-- GATHERING FIELDS -->
+            <field name="recordedBy" term="http://rs.tdwg.org/dwc/terms/recordedBy">
+                <path>gatheringEvent.gatheringPersons.0.fullName</path>
+            </field>
+            <!-- eventDate is gemapt naar "dateBegin" en niet naar de range "dateBegin/dateEnd" -->
+            <field name="eventDate" term="http://rs.tdwg.org/dwc/terms/eventDate">
+                <calculator>
+                    <java-class>CalendarDateCalculator</java-class>
+                    <arg>gatheringEvent.dateTimeBegin</arg>
+                </calculator>
+            </field>
+            <field name="eventTime" term="http://rs.tdwg.org/dwc/terms/eventTime">
+                <calculator>
+                    <java-class>GatheringEventTimeCalculator</java-class>
+                </calculator>
+            </field>
+            <field name="verbatimEventDate" term="http://rs.tdwg.org/dwc/terms/verbatimEventDate">
+                <path>gatheringEvent.dateText</path>
+            </field>
+            <field name="fieldNotes" term="http://rs.tdwg.org/dwc/terms/fieldNotes">
+                <path>notes</path>
+            </field>
+            <field name="behavior" term="http://rs.tdwg.org/dwc/terms/behavior">
+                <path>gatheringEvent.behavior</path>
+            </field>
+            <field name="sex" term="http://rs.tdwg.org/dwc/terms/sex">
+                <path>sex</path>
+            </field>
+            <field name="lifeStage" term="http://rs.tdwg.org/dwc/terms/lifeStage">
+                <path>phaseOrStage</path>
+            </field>
+            <field name="preparations" term="http://rs.tdwg.org/dwc/terms/preparations">
+                <path>preparationType</path>
+            </field>
+            <!-- OTHER FIELDS -->
+            <field name="references" term="http://purl.org/dc/terms/references">
+                <path>unitGUID</path>
+            </field>
+            <field name="Associated Taxa" term="http://rs.tdwg.org/dwc/terms/associatedTaxa">
+                <calculator>
+                    <java-class>AssociatedTaxaNameCalculator</java-class>
+                </calculator>
+            </field>
+            <field name="rightsHolder" term="http://purl.org/dc/terms/rightsHolder">
+                <path>owner</path>
+            </field>
+            <field name="license" term="http://purl.org/dc/terms/license">
+                <path>license</path>
+            </field>
+        </mapping>
+    </entity>
+    <entity name="multimedia">
+        <data-source>
+            <document>MultiMediaObject</document>
+            <query-spec>
+                <conditions>
+                    <condition>
+                        <field>collectionType</field>
+                        <operator>EQUALS_IC</operator>
+                        <value>Wildlife sounds - Grasshoppers</value>
```

Comment:
@jeroencreuwels deze query is niet goed. Je haalt hier de sprinkhaan multimedia binnen om te koppelen aan de vleermuis specimen data!

Reply:
Had vergeten de Multimedia aan te passen. Nu gaat het goed